example/run.ts

```diff
@@ -97,3 +97,24 @@ monitored(
     },
     {logErrorAsInfo: true}
 ).catch(() => {});
+
+// Increment
+increment('metric1');
+
+// Increment with type restriction
+type Color = 'red' | 'orange' | 'green';
+increment<Color>('red');
+
+// monitor method using decorator
+class Person {
+    firstName: string = 'Jon';
+    lastName: string = 'Doe';
+
+    @monitorMethod('person', {logErrorAsInfo: true})
```

Comment:
Why not monitored?

Reply:
it is already taken :/

Reply:
I'm opposed, you'd easily have cases where you have both in the same file